src/step-input.scss

```diff
@@ -84,6 +84,15 @@ $block: #{$fd-namespace}-step-input;
     }
   }
 
+  &__container {
+    display: flex;
+    align-items: center;
```

Comment:
Hi @manu-kr , there are checkboxes in the PR template that are put there with a purpose, not just to waste people's time to check them when they open a PR. One of these checks is about mixins:

<img width="762" alt="Screen Shot 2021-04-29 at 11 10 33 AM" src="https://user-images.githubusercontent.com/39598672/116574476-bef58e00-a8db-11eb-9986-55df2048cead.png">

In `_mixins.scss` file there's a mixin called `fd-flex-vertical-center` which is basically this:
<img width="489" alt="Screen Shot 2021-04-29 at 11 13 33 AM" src="https://user-images.githubusercontent.com/39598672/116574660-f401e080-a8db-11eb-9e89-cf34f8ab3592.png">

You can use it in your code.

Reply:
Hi @InnaAtanasova , Sorry i have missed this. I have reused this now.